api-server/src/sensors/model.rs

```diff
@@ -3,22 +3,36 @@ use crate::error_handler::CustomError;
 use crate::schema::sensors;
 use crate::sensor_types::{SensorTypesModel};
 use crate::stations::{StationsModel};
+use crate::measurements::MeasurementsModel;
 
 use chrono::{Local, NaiveDateTime};
 use diesel::prelude::*;
 use serde::{Deserialize, Serialize};
-use uuid::Uuid;
+use uuid:: Uuid;
+
+#[derive(Deserialize, Serialize)] 
+pub struct Sensor {
+    pub id: uuid::Uuid,
+    pub alias: String,
+    pub label: String,
+    pub type_id: uuid::Uuid,
+    pub station_id: uuid::Uuid,
+    pub created_at: chrono::NaiveDateTime,
+    pub updated_at: chrono::NaiveDateTime,
+    pub measurements: Vec<MeasurementsModel>
+}
 
 #[derive(Serialize, Deserialize, AsChangeset, Insertable)]
 #[table_name = "sensors"]
 pub struct SensorsChangeset {
     pub alias: String,
     pub label: String,
-    pub type_id: Uuid,
-    pub station_id: Uuid
+    pub type_id: uuid::Uuid,
+    pub station_id: uuid::Uuid
```

Comment:
We imported Uuid so that you can reference it directly as `Uuid` without the need for the `uuid::` prefix.